packages/helpers/classes/attachment.js

```diff
@@ -39,17 +40,29 @@ class Attachment {
     data = toCamelCase(data);
 
     //Extract properties from data
-    const {content, filename, type, disposition, contentId} = data;
+    const {content, filename, type, disposition, contentId, file} = data;
```

Comment:
I'm worried that `file` vs. `filename` will be a bit confusing. Maybe instead of passing a file object, we can have two separate new settings `filePath` and `fileEncoding`. 

(The `filename` setting should probably be deprecated and introduce a new one `fileName` to be consistent)